src/clang-c-frontend/clang_c_convert.cpp

```diff
@@ -1,14 +1,17 @@
 // Remove warnings from Clang headers
-#pragma GCC diagnostic ignored "-Wstrict-aliasing"
 #pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wstrict-aliasing"
 #pragma GCC diagnostic ignored "-Wunused-parameter"
 #include <clang/AST/Attr.h>
+#include <clang/AST/Expr.h>
 #include <clang/AST/QualTypeNames.h>
+#include <clang/AST/Type.h>
 #include <clang/Index/USRGeneration.h>
+#include <clang/Frontend/ASTUnit.h>
+#pragma GCC diagnostic pop
+
```

Comment:
This approach should check for the compiler since we are going to support all major compilers. MSVC will throw a warning for this.